CMakeLists.txt

```diff
@@ -49,6 +49,185 @@ if(GNU_GCC)
   add_compile_options(-pipe)
 endif()
 
+option(WARNINGS_PEDANTIC "Let the compiler show even more warnings" OFF)
+if(MSVC)
+  if(WARNINGS_PEDANTIC)
+    add_compile_options(/W4)
```

Comment:
Do we need to change this as well? I don't think it makes sense to increase the warning level for Mixxx dependencies.

Reply:
We may discuss it.

My idea was that we are responsible for the code in the lib folders and we should know if there is something fishy going on.

We have already done warning in the lib folders. The question is, if we want to care about it on the long run or not ...

What do you think?

Reply:
I don't see why we should start to fix warnings in our external dependencies when we just overwrite the whole directory next time we update the version anyway tbh. Sound like a lot of unnecessary work.

Reply:
OK I have reverted this change. 
However I do think that we should look at warnings when compiling external dependencies. 
They may reveal an issue that only happens in our build context. 
Than we write a patch againts the upstream source. So that it persist for future updates.